src/repl.jl

```diff
@@ -189,11 +189,13 @@ function julia_prompt(state::DebuggerState)
             return false
         end
         command = String(take!(buf))
-        @static if VERSION >= v"1.2.0-DEV.253"
-            response = _eval_code(active_frame(state), command)
+        response = _eval_code(active_frame(state), command)
+        @static if VERSION >= v"1.11.5"
+            fetch(@async REPL.print_response(state.repl, response, true, true))
```

Comment:
is there a reason this isn't using `@spawn` instead?

Reply:
No reason. Did it work like this?

Reply:
yeah. It works both with `@async` and `@spawn`

Reply:
Great. Updated. Needs CI to be run, if you have perms.

Reply:
I don't. @KristofferC can you run CI?